server/jetstream.go

```diff
@@ -228,7 +228,7 @@ func (s *Server) restartJetStream() error {
 		MaxStore:  opts.JetStreamMaxStore,
 	}
 	s.Noticef("Restarting JetStream")
-	err := s.enableJetStream(cfg)
+	err := s.EnableJetStream(&cfg)
```

Comment:
This change is needed because otherwise the path was being changed from `path/store/jetstream` to `path/store/`

Reply:
Currently, when JS starts with dynamic limits, it stores files at

```
store.Dir + "/jetstream"
```

but when limits are set, it does not append `jetstream`.